lib/tapioca/cli.rb

```diff
@@ -149,6 +154,7 @@ def dsl(*constant_or_paths)
         exclude: options[:exclude],
         file_header: options[:file_header],
         tapioca_path: TAPIOCA_DIR,
+        skip_constant: options[:skip_constant],
```

Comment:
Should we check that `requested_constants` and `skip_constant` do not conflict? Or maybe add a note about which one takes precedence?